jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/util/GoTimeUtilTest.java

```diff
@@ -43,9 +44,9 @@ public void testConversion() {
 
     @Test
     public void testEmpty() {
-        Assertions.assertThat(GoTimeUtil.durationSeconds(null)).isEqualTo(Optional.empty());
-        Assertions.assertThat(GoTimeUtil.durationSeconds("")).isEqualTo(Optional.empty());
-        Assertions.assertThat(GoTimeUtil.durationSeconds(" ")).isEqualTo(Optional.empty());
+        assertEquals(Optional.empty(), GoTimeUtil.durationSeconds(null));
+        assertEquals(Optional.empty(), GoTimeUtil.durationSeconds(""));
+        assertEquals(Optional.empty(), GoTimeUtil.durationSeconds(" "));
```

Comment:
Thanks a lot for your PR :+1: . Could you please separate these three assertions into three separate tests?

Reply:
Thanks. That was done.